fe/fe-core/src/main/java/org/apache/doris/statistics/ColumnStats.java

```diff
@@ -76,6 +76,22 @@ public class ColumnStats {
     private LiteralExpr minValue;
     private LiteralExpr maxValue;
 
+    public ColumnStats(ColumnStats other) {
+        this.ndv = other.ndv;
+        this.avgSize = other.avgSize;
+        this.maxSize = other.maxSize;
+        this.numNulls = other.numNulls;
+        if (other.minValue != null) {
+            this.minValue = (LiteralExpr) other.minValue.clone();
```

Comment:
Why clone?

Reply:
To avoid coder change the child's stats unexpectedly when writing the stats logic for other internal node of the plan tree.